ldb/admin.py

```diff
@@ -60,10 +67,117 @@ def __init__(self, *args, **kwds):
         self.fields['living_with'].queryset = Person.objects.order_by('surname', 'firstname')
 
 
+class PersonResource(resources.ModelResource):
+    def before_import(self, dataset, using_transactions, dry_run, **kwargs):
+        if 'id' not in dataset.headers:
+            dataset.lpush_col([None for row in range(len(dataset))], header='id')
+
+    def init_instance(self, row=None):
+        instance = self._meta.model()
+        try:
+            student = Student.objects.get(
+                student_number=str(row['student__student_number'])  # Cast to string as it is saved like this in the DB
+            )
+        except Student.DoesNotExist:
+            student = Student()
+
+        try:
+            member = Member.objects.get(
+                person_id=row['id']
+            )
+        except Member.DoesNotExist:
+            member = Member()
+
+        instance.member = member
+        instance.student = student
+        return instance
+
+    def get_or_init_instance(self, instance_loader, row):
+        """
+        Either fetches an already existing instance or initializes a new one.
+        """
+        try:
+            instance = Person.objects.get(
+                Q(netid=str(row['netid'])) | Q(student__student_number=str(row['student__student_number']))
```

Comment:
Hmmm.. This way it could raise a `MultipeObjectsReturned` (in the case we have two different people with this netid and student number). But maybe that isn't a real problem since exceptions will be shown to the person importing the data and it should be fixed by hand anyway.

Reply:
Hmm let's think about this one. I have tested updating information for someone with an already known netid and studentnumber, and the variations of that. It updated the record correctly. Both student and netid have `unique=True`, so is it possible that there are multiple objects with the same netid or student number? In that case the record is simply updated. I would hope that the TU does not give out identical student numbers or netids.

Reply:
This would happen when you have the same person twice in the database, once with the netid entered and once with the student number. 
The underlying conflict should be solved by hand. So maybe we shouldn't try to fix this situation. However, we could catch the `Person.MultipleObjectsReturned` and give a more user friendly error message.